frictionless/resource.py

```diff
@@ -576,13 +576,21 @@ def header(self):
         """
         return self.__header
 
```

Comment:
Remove __setattr__ method as it's not implemented using python property setters?